### PR TITLE
Add PVP Api docs validation test to alembic CI 

### PR DESCRIPTION
**Pull request by windxu88**
**Purpose of this PR:**
Add Package Validation Profiles (PVP)'s API docs validation test to Alembic package

**JIRA ticket:**
[ABC-316](https://jira.unity3d.com/browse/ABC-316) CI: Add Package Validation Profiles (PVP)'s API docs validation test to Alembic package

### CODE REVIEW

File: .yamato/upm-ci.yml [hunk 1]
```diff
@@ -59,6 +59,34 @@ bizarro_standalone_platforms:
   #  runtime: stadia
 
 ---
+docs-validate:
```
Comment (clusty) [inline .yamato/upm-ci.yml:62]: I would add this as a dependency on Test_Trigger ? (the one triggered by every PR)

Comment (windxu88) [reply .yamato/upm-ci.yml:62]: Good catch. Just added it to test_trigger:)

**Review by clusty — APPROVED** [review]
YEEEY

**Comment by windxu88** [comment]
@RoseHirigoyen Could you please also review this PR? It is all green now. Thanks:)
@clusty Do I need to integrate this fix into different branches, maybe release/2.2?

**Comment by clusty** [comment]
I would let it go for now. We are unlikely to do any release in 2.2 in the near future (and those won't include any API surely)

**Review by RoseHirigoyen — APPROVED** [review]

**Pull request merged**